app/modules/asset_groups/models.py

```diff
@@ -383,43 +383,42 @@ def detected(self, job_id, data):
 
             for annot_id in range(len(results[asset_id])):
                 annot_data = results[asset_id][annot_id]
-                uuid = annot_data.get('uuid', None)
```

Comment:
FYI, this little gotch  caused the uuid.UUID call in line 380 to go bananas the second time round the loop